components/job-orchestration/job_orchestration/scheduler/constants.py

```diff
@@ -7,21 +11,47 @@ class QueueName:
     SEARCH = "search"
 
 
-class JobStatus:
+class CompressionJobStatus:
     SCHEDULING = 'SCHEDULING'
     SCHEDULED = 'SCHEDULED'
     SUCCEEDED = 'SUCCEEDED'
     FAILED = 'FAILED'
 
 
-class TaskUpdateType:
+class CompressionJobCompletionStatus(IntEnum):
+    SUCCEEDED = 0
+    FAILED = 1
+
+
+class CompressionTaskUpdateType:
     COMPRESSION = 'COMPRESSION'
     SEARCH = 'SEARCH'
 
 
-class TaskStatus:
+class CompressionTaskStatus:
     SUBMITTED = 'SUBMITTED'
     SCHEDULED = 'SCHEDULED'
     IN_PROGRESS = 'IN_PROGRESS'
     SUCCEEDED = 'SUCCEEDED'
     FAILED = 'FAILED'
+
+
+# When adding new states always add them to the end of this enum
+# and make necessary changes in the UI, Search Scheduler, and Reducer
+class SearchJobStatus(IntEnum):
+    PENDING = 0

```

Comment:
I wonder if we should make the CompressionJobStatus and CompressionTaskStatus classes consistent with SearchJobStatus? I.e.,

* `SCHEDULING/SUBMITTED` -> `PENDING`
* `SCHEDULED` -> `RUNNING
* Use an IntEnum rather than strings